src/cpp/session/modules/NotebookData.R

```diff
@@ -70,16 +70,19 @@
   )
 })
 
-.rs.addFunction("runSqlForDataCapture", function(query, connectionName, outputFile)
+.rs.addFunction("runSqlForDataCapture", function(query, outputFile, options)
 {
   # precreate directories if needed
   dir.create(dirname(outputFile), recursive = TRUE, showWarnings = FALSE)
 
-  conn <- get(connectionName, envir = globalenv())
+  maxRows <- if (is.null(options$max.print)) 1000 else as.numeric(options$max.print)
+  maxRows <- if (is.null(options$sql.max.print)) maxRows else as.numeric(options$sql.max.print)
```

Comment:
This is actually a knitr option not a chunk option (it's there so that the html_notebook format can force knitr to show 1000 rows rather than 10 rows by default). Net: we can remove this line.